CovertInfiltration/Src/CovertInfiltration/Classes/X2InfiltrationMod_DefaultModifiers.uc

```diff
@@ -0,0 +1,84 @@
+//---------------------------------------------------------------------------------------
+//  AUTHOR:  NotSoLoneWolf
+//  PURPOSE: This class contains all the X2InfiltrationModTemplates
+//           required for the mod, covering all basegame armors and weapons
+//---------------------------------------------------------------------------------------
+//  WOTCStrategyOverhaul Team
+//---------------------------------------------------------------------------------------
+
+class X2InfiltrationMod_DefaultModifiers extends X2InfiltrationMod;
+
+static function array<X2DataTemplate> CreateTemplates()
+{
+	local array<X2DataTemplate> Templates;
+	local X2InfiltrationMod Base;
+	Base = new class'X2InfiltrationMod';
+	
+	//Templates.AddItem(Base.static.BuildArmorTemplate(name DataName, int Weight, optional float Mult, optional name MultCat));
+	//Templates.AddItem(Base.static.BuildWeaponTemplate(name DataName, int Weight, optional float Mult, optional name MultCat));
+	//Templates.AddItem(Base.static.BuildCustomTemplate(name DataName, optional int Modifier, optional float Mult, optional name MultCat));
+	
+	Templates.AddItem(Base.static.BuildArmorTemplate('KevlarArmor_InfilMod', 3));

```

Comment:
Can't you just do this?

```
Templates.AddItem(BuildArmorTemplate('KevlarArmor_InfilMod', 3));
```

Reply:
No, I would like to have the DataSet class contain the functions for ease of access by other mods.

Reply:
Yes, I agree. I meant that you can call base class methods without the prefix, just by name